app/api/controllers/RequestHandler.scala

```diff
@@ -109,19 +109,27 @@ object RequestHandler {
           message = s"[${ctx.endpointLogContext.controllerName}][${ctx.endpointLogContext.endpointName}] " +
             s"with correlationId : ${ctx.correlationId}")
 
-        val result =
-          for {
-            parsedRequest   <- EitherT.fromEither[Future](parser.parseRequest(rawData))
-            serviceResponse <- EitherT(service(parsedRequest))
-          } yield doWithContext(ctx.withCorrelationId(serviceResponse.correlationId)) { implicit ctx: RequestContext =>
-            handleSuccess(rawData, parsedRequest, serviceResponse)
-          }
-
-        result.leftMap { errorWrapper =>
-          doWithContext(ctx.withCorrelationId(errorWrapper.correlationId)) { implicit ctx: RequestContext =>
-            handleFailure(errorWrapper)
-          }
-        }.merge
+        val maybeGovTestScenario = ctx.hc.otherHeaders.find(header => header._1 == "Gov-Test-Scenario").map(headers => headers._2).toString
+
+        if (maybeGovTestScenario == "REQUEST_CANNOT_BE_FULFILLED") {
+          Future.successful(ResultWrapper(422, Some(Json.toJson("Custom (will vary depending on the actual error)"))).asResult)
```

Comment:
Probably better msg simply "Request cannot be fulfilled", but Ben can confirm

Reply:
Ben said "Custom (will vary **in production** depending on the actual error)" would be best as "Request cannot be fulfilled" is already stated from the code and plan is for the message to be dependent on the downstream error that caused it.